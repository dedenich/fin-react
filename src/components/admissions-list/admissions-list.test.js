import React from 'react';
import renderer from 'react-test-renderer';

import AdmissiomsList from './admissions-list.jsx';


it(`AdmissiomsList renders correctly`, () => {
  const tree = renderer
  .create(
      <AdmissiomsList
        admissionsList={[]}
        isContentShown={false}
        collectedAmount={4}
      />
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
