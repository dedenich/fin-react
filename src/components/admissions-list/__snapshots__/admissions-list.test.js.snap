// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdmissiomsList renders correctly 1`] = `
<li>
  Всего
  : 
  <span>
    4.00
     BYN
  </span>
</li>
`;
